src/style.css

```diff
@@ -1,3 +1,85 @@
+* {
```

Comment:
Avoid using the universal selector '*' for resetting margins and paddings as it can lead to performance issues. Consider using a more specific selector or a CSS reset/normalize stylesheet.